_rules/html-page-lang-matches-default-ucwvc8.md

```diff
@@ -217,6 +221,24 @@ This page has `lang` attribute value of `nl` (Dutch), which does not matche the
 </html>
 ```
 
+#### Failed Example 5
+
+This page has `lang` attribute value of `nl` (Dutch), which does not match the [default language of the page][default page language]. The default language is English because the accessible name of the `img` element is English. The `lang` attribute on the `p` element is effectively ignored.
```

Comment:
```suggestion
This page has a `lang` attribute value of `nl` (Dutch), which does not match the [default language of the page][default page language]. The default language is English because the accessible name of the `img` element is English. The `lang` attribute on the `p` element is effectively ignored.
```